### PR TITLE
Escaping fixed

### PR DESCRIPTION
**Pull request by ondrejcech**

### CODE REVIEW

File: src/PoCompiler.php [hunk 1]
```diff
@@ -273,7 +265,8 @@ protected function cleanExport($string)
      */
     private function wrapString($value)
     {
-        $wrapped = \wordwrap($value, $this->wrappingColumn, " \n");
-        return \explode("\n", $wrapped);
+        $value = str_replace("\n", chr(13) . chr(28), $value);
```
Comment (raulferras) [inline src/PoCompiler.php]: I feel like `chr(13)`, `chr(28)` and similar are hard to read. At least to know what characters they are representing.  
I would create variables for them with auto-explanative names.  
Example:
```
$carriageReturn = chr(13);
$somethingExplanatori = chr(28);
```

File: src/PoCompiler.php [hunk 2]
```diff
@@ -254,17 +254,9 @@ protected function buildProperty($property, $value, $obsolete = false)
      */
     protected function cleanExport($string)
     {
-        $quote   = '"';
-        $slash   = '\\';
-        $newline = "\n";
+        $string = sprintf('"%s"', addcslashes($string, "\42\134")); // " and \
```
Comment (raulferras) [inline src/PoCompiler.php]: As explained below, `\42\134` is hard to understand, what and why this exact value?

File: src/PoCompiler.php (hunk 1 above)
Comment (ondrejcech) [reply src/PoCompiler.php]: You're totally right. I've fixed that. See latest commit.

File: src/PoCompiler.php (hunk 2 above)
Comment (ondrejcech) [reply src/PoCompiler.php]: You're totally right. I've fixed that. See latest commit.

**Comment by raulferras** [comment]
As this PR introduces a change in behaviour, I will bump version to 6.0.0

**Pull request merged**